Portal/src/Datahub.Portal/wwwroot/css/wet-boew/theme-wet-boew/css/theme.css

```diff
@@ -0,0 +1,1739 @@
+*:focus {
+    outline: 2px solid black !important;
+    outline-offset: -2px !important;
+}
+
+.mud-checkbox-input:hover {
+    outline: 2px solid black !important;
+    outline-offset: -2px !important;
+}
+
+.mud-button:hover {
+    outline: 2px solid black !important;
+    outline-offset: -2px !important;
+}
+
+
+.mud-nav-link:hover {
+    outline: 2px solid black !important;
+    outline-offset: -2px !important;
+}
+
+.mud-link {
+    text-decoration: underline !important;
+}
```

Comment:
Can we move this to it's own file instead of including 1715 commented css lines

Reply:
The commented css files are temporarily commented, theyre actually part of the forms work so theyll be uncommented in 1-2 git commits

Reply:
Ah okay, can you still separate them into different files? Something like `mud-accessibility-fixes.css`